New version: PALEOboxes v0.22.4

JuliaRegistrator opened the pull request:
- Registering package: PALEOboxes
- Repository: https://github.com/PALEOtoolkit/PALEOboxes.jl
- Created by: @sjdaines
- Version: v0.22.4
- Commit: 6e0761da8b7182fc3d5a6dc50eb9cafb14e4d954
- Reviewed by: @sjdaines
- Reference: https://github.com/PALEOtoolkit/PALEOboxes.jl/commit/6e0761da8b7182fc3d5a6dc50eb9cafb14e4d954#commitcomment-153190690
- Description: Model coupler for the PALEO model framework
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6430efa1f9ea633acf4e2bf88f21c78a7862af89e222f799ae12256823cb3e3b001acfc624978ebeb0fdde0da447a963a69d5bf914712802ba68776b39ce2f3b5079a00826e5cf39f33ffffd5bc8d01e6f96fe6630cef3d5566dc339a36d855487d1953e8121aa44725086dcc0f527fe24ce12b4050efc7a7e07ea15f686684686aaf638fe75f83426aa555df430ce15836ed61dc39263d349b1a9d1c5a56cd0cb9961cdf4d43559967fa67a63e10447c -->

sjdaines commented:
[noblock] This PR has apparently got stuck (has been sitting unmerged for a day) despite the 'Automerge guidelines all met' message.

Try add this comment to see if it might restart the merge process

JuliaRegistrator committed:
New version: PALEOboxes v0.22.4

UUID: 804b410e-d900-4b2a-9ecd-f5a06d4c1fd4
Repo: https://github.com/PALEOtoolkit/PALEOboxes.jl.git
Tree: b3a2436209229ab0332f30c5969117eafbf7f87d

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
